About.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML course!</title>
+
+    <link rel="stylesheet" href="Styles.css">
+    <link rel="stylesheet" href="Stylesdesktop.css">
+    <style>
+        @import url('https://www.dafont.com/es/cocogoose.font');
+    </style>
+
+                
+
+</head>
+<body>
+    <section class="header">
+        <div>
+            <ul class="pre-nav-bar desktop">
+                <li class="socialMedia"><a class="icon" href="https://github.com/Kakalanp"><img src="Images/GitHub.png" alt=""></a></li>
+                <li class="socialMedia"><a class="icon" href="https://www.linkedin.com/in/joaquín-garrido-lecca-zanetti-623583204/"><img src="Images/LinkedIn.png" alt=""></a></li>
+                <li class="socialMedia"><a class="icon" href="https://twitter.com/LeccaJoaquin"><img src="Images/twitter.png" alt=""></a></li>
+                <li><a class="link">English</a></li>
+                <li><a class="link">My Page</a></li>
+                <li><a class="link">Logout</a></li>
+            </ul>
+        </div>
+
+        <nav class="nav-bar">
+            <a href="index.html" class="logo desktop"><img class="logo" src="Images/Logo.png" alt="Logo"></a>
+            <ul class="menu">
+                <li><a class="active" href="About.html">About</a></li>
+                <li><a>Program</a></li>
+                <li><a>Join</a></li>
+                <li><a>Sponsors</a></li>
+                <li><a>News</a></li>
+                <li><a class="cc-campaign">CC Campaign</a></li>
+            </ul>
+            <a class="hamburger mobile" href="#">≡</a>
+        </nav>
```

Comment:
- [ ] Good job implementing the navbar feature, it's working perfect, but there is a small issue : 
 - When I am on the `About` page in the mobile version I can't redirect to the home page using the navbar, Kindly try to work on this issue, as you can in every website those little details are so important 
 
![about](https://user-images.githubusercontent.com/80895497/152261494-625c859e-e3e8-4fc9-9fe5-60feb6469dda.png)

Reply:
The last version now does this, moving the logo inside the mobile menu on-open: https://github.com/Kakalanp/Capstonproj-Conference-page/blob/0fe222a7dd22cd537af97f720b6aa0831dcd83e3/Js/script.js#L71-L73

And removing it on-close: https://github.com/Kakalanp/Capstonproj-Conference-page/blob/0fe222a7dd22cd537af97f720b6aa0831dcd83e3/Js/script.js#L51-L54

This is how it looks in the page now: 
![image](https://user-images.githubusercontent.com/78335447/152381848-0cfee2b9-762c-4f32-87be-6ec59c435f59.png)